acmw-db-app/components/GHCVolunteerForm.jsx

```diff
@@ -0,0 +1,57 @@
+import SubmitButton from './FormComponents/SubmitButton'
+import TextField from './FormComponents/TextField'
+import SelectInput from './FormComponents/SelectInput'
+import MultiSelectInput from './FormComponents/MultiSelectInput'
+import styles from '../styles/components/CompanyForm.module.css'
+import {useState} from 'react'
+
+const GHCVolunteerForm = (props) => {
+
+    // eventually connect this to actual GHC scholarship awardees from database
+    // Can implement a search similar to "company search component" but for members
+    const GHCNames = ["Milly Mason", "Jing George", "Angela Li", "Morgan Zahner", "Sara Miskus", "Amy Huang"]
+
+    const [GHCName, setGHCName] = useState("")
+    const [volunteerEventName, setVolunteerEventName] = useState("")
+    const [numHours, setNumHours] = useState(0)
+    const [hourError, setHourError] = useState("")
+
+    const onSubmit = () => {
+        // replace later with an api call to POST data
+        
+    }
+
+    const validateHours = (value) => {
```

Comment:
looks great but could you also apply this to the name too? just like require their is something in the name and not blank